core/src/main/java/bisq/core/app/BisqEnvironment.java

```diff
@@ -206,7 +206,7 @@ public static boolean isDaoActivated(Environment environment) {
             torRcFile, torRcOptions, externalTorControlPort, externalTorPassword, externalTorCookieFile,
             socks5ProxyHttpAddress, useAllProvidedNodes, numConnectionForBtc, genesisTxId, genesisBlockHeight, referralId, daoActivated;
 
-    protected final boolean externalTorUseSafeCookieAuthentication;
+    protected final boolean externalTorUseSafeCookieAuthentication, torStreamIsolation;
```

Comment:
Should torStreamIsolation be by default turned on (true)?

Reply:
I have been thinking about that as well.
- However, in the past, we had some issues with `SecureRandom` on Linux taking an extraordinary amount of time to produce its first output (> 2 minutes) which could only be sped up by changing something in the OS.
- Furthermore, creating a random string and a new Socket every time takes time.
- Last but not least, I am not entirely sure if changing the socket everytime does effect network performance to the worse.

I added an "experimental!" to the option so we can life the `--streamIsolation` for a little while and see if we encounter any unforeseen issues. If, one day, we are convinced that everything is fine, we can use the feature by default.

Or should we do it the other way around and provide an off-switch to the feature?

Reply:
Yes, good points. Is SecureRandom really needed at all here? I have not followed closely the context and motivation behind it but to keep it turned off by default is definitely better.